README.md

```diff
@@ -0,0 +1,47 @@
+## Drink This
+### Live at
+
+https://https://drink-this-frontend.herokuapp.com/
+
+## About this Project
+
+Describe the project
```

Comment:
Maybe we should put something like...

"Drink This is a cocktail recommendation app that provides cocktail recommendations to the user. In the front end of this app, we used ReactJS to provide a clean feel. Movement between pages is smooth and seamless, as opposed to rendering views that may take time to load. Another look and feel to this is the frontend interaction with the backend. We displayed simple thumbnails of the cocktails consumed from the CocktailDB API from the backend."